### PR TITLE
Fix CreditUnion spider

### PR DESCRIPTION
**Pull request by jancallewaert**
The lat and lon of the Credit Unions have disappeared from their site. I don't know if we allow a spidered POI without lat/lon.

### CODE REVIEW

**Comment by Cj-Malone** [comment]
We do collect things even if we can't get coordinates. They could be geo coded by a data consumer, or potentially at some point by ATP.

**Comment by Cj-Malone** [comment]
Would you mind replacing the state array with `GeonamesCache().get_us_states()` please.

File: locations/spiders/creditunion.py [hunk 1]
```diff
@@ -107,24 +107,29 @@ def parse_bank(self, bank):
         except IndexError:
             phone = ""
 
-        href = bank.xpath('.//div[@class="location-results__share"]/div/a/@href').extract_first()
-        lat, lon = re.search(r"lat=(.*?)&lng=(.*?)&", href).groups()
         name = bank.xpath(".//h3/text()").extract_first()
-        ref = name + "-" + "-".join(address[0].split(" "))
+        ref = "-".join(name.split(" ")) + "-" + "-".join(address[0].split(" "))
```
Comment (iandees) [inline locations/spiders/creditunion.py:111]: All the Places generates IDs based on a hash of the `ref` and the spider name, so we prefer to keep the `ref` consistent where possible. Unless there's a specific reason you made this change, can we roll this particular line back to keep the name consistent?

```suggestion
        ref = name + "-" + "-".join(address[0].split(" "))
```

Comment (JanCallewaert-TomTom) [reply locations/spiders/creditunion.py]: Fixed

**Comment by Cj-Malone** [comment]
Thank you

**Pull request merged**